src/consensus/blockstore.rs

```diff
@@ -267,10 +283,47 @@ mod tests {
         (sk, BlockstoreImpl::new(Arc::new(epoch_info), tx))
     }
 
+    #[tokio::test]
+    async fn store_slice() -> Result<()> {
```

Comment:
tangential: do we prefer tests to return a `Result` or tests that just assert the conditions they want and return nothing?  I see both flavours in the code base.  I think I have been adding mainly the second type so for.

Reply:
I just return `Result` here and in other places to be able to use the `?` operator instead of `unwrap()` for errors that are not part of the main test logic